common/src/main/java/org/opensearch/ml/common/connector/MLPostProcessFunction.java

```diff
@@ -18,38 +18,62 @@ public class MLPostProcessFunction {
 
     public static final String COHERE_EMBEDDING = "connector.post_process.cohere.embedding";
     public static final String OPENAI_EMBEDDING = "connector.post_process.openai.embedding";
-
+    public static final String BEDROCK_EMBEDDING = "connector.post_process.bedrock.embedding";
     public static final String DEFAULT_EMBEDDING = "connector.post_process.default.embedding";
 
     private static final Map<String, String> JSON_PATH_EXPRESSION = new HashMap<>();
 
-    private static final Map<String, Function<List<List<Float>>, List<ModelTensor>>> POST_PROCESS_FUNCTIONS = new HashMap<>();
+    private static final Map<String, Function<List<?>, List<ModelTensor>>> POST_PROCESS_FUNCTIONS = new HashMap<>();
```

Comment:
Why do we need `<?>`? What other type are we expecting here? In the tensor datatype if we are ensuring `dataType(MLResultDataType.FLOAT32)` why do we need to change here?

Reply:
BedRock has different response with other platforms, BedRock doesn't support `list of string` input so the result is a single dimension list instead of double dimension `list of float`, to adapt to this difference, we need to change the type to a generic type here.